### PR TITLE
bg, fg, jobs: add Russian translation

### PR DESCRIPTION
**Pull request by trolzen**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.ru/common/bg.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# bg
+
+> Возобновляет работу приостановленного задания (например, с помощью `Ctrl + Z`), и оставляет его работать в фоне.
+> Больше информации: <https://manned.org/bg>.
+
+- Возобновить работу последнего приостановленного задания и продолжить его выполнение в фоне:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/bg.md:6]: I am not sure about Russian terminology. Term `задание` confuses me.

**Review by EmilySeville7cfg — APPROVED** [review]
This PR should be reviewed by someone who knows Russian terminology related to job management in Bash.

**Comment by kbdharun** [comment]
> This PR should be reviewed by someone who knows Russian terminology related to job management in Bash.

Are there other Russian reviewers in tldr?

> If so can you request a review from them?

File: pages.ru/common/bg.md (hunk 1 above)
Comment (trolzen) [reply pages.ru/common/bg.md:6]: @EmilySeville7cfg The term `задание` is well-established translation for the term `job` in this context. See for example [Russian man page for Bash on OpenNET.ru](https://www.opennet.ru/man.shtml?topic=bash&category=1&russian=0) (grep for section "УПРАВЛЕНИЕ ЗАДАНИЯМИ") .

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**